src/onegov/form/parser/grammar.py

```diff
@@ -603,7 +603,7 @@ def marker_box(characters: str) -> ParserElement:
     #       If pricing wasn't optional this would be easy...
     label = Regex(
         # a sequence of words (that can't start with brackets)
-        rf'(?P<label>{text_without_re(characters + "()")}(?: ?{word_re})*?) *'
+        rf'(?P<label>{text_without_re(characters + "()")}(?: *{word_re})*?) *'
```

Comment:
In hindsight I'm not convinced this is right call, since all the other labels still require single spaces, so I think my other suggested change is more appropriate, it mirrors what would've happened previously, i.e. it cuts off anything it doesn't recognize as part of the label.